plugins/cross-tab-copy-paste/package.json

```diff
@@ -45,7 +44,7 @@
     "blockly": "^8.0.0"
   },
   "peerDependencies": {
-    "blockly": "^8.0.0"
```

Comment:
I think the change your thinking of is the one that changed the return type of paste. If so, that's for fixing keyboard nav so that it doesn't access internal properties.

Reply:
Confirming that I did not add anything to core for this plugin and I tested it works with v7 before merging.